PRS13001-14000/PRS13403Krestina.xml

```diff
@@ -38,29 +38,35 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             </langUsage>
 <textClass>
 <keywords>
+   <term key="saint"/>
 <term key="martyr"/>
 </keywords>
 </textClass>
         </profileDesc>
         <revisionDesc>
             <change who="MKr" when="2021-04-17">Created entity</change>
+            <change who="MKr" when="2025-02-17">Added relation, link to Wikidata</change>
         </revisionDesc>
     </teiHeader>
     <text>
         <body>
             <listPerson>
-                <person sex="2">
-                    <persName xml:lang="gez" xml:id="n1">ክርስጢና፡</persName>
+                <person sameAs="wd:Q237882" sex="2">
+                     <persName xml:lang="gez" xml:id="n1">ክርስጢና፡</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Krǝsṭinā</persName>
                     <persName xml:lang="en" corresp="n2">Christine</persName>
                     <persName xml:lang="gez" xml:id="n3">ክርስጢና፡ ወለተ፡ ርባኖስ፡</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n3">Krǝsṭinā walatta Rǝbānos</persName>
+                    <persName xml:lang="en" xml:id="n4">St Christina of Bolsena</persName>
```

Comment:
Thank you! for non-Ethiopian saints it may be better to have the standard English name as n1